docs/src/contribute/contribute-core-rule.md

```diff
@@ -0,0 +1,141 @@
+---
```

Comment:
Can we just name the file `core-rules.md`? Otherwise the URL will say `contribute/contribute-core-rule` vs `contribute/core-rules`

Reply:
done!